src/sparsezoo/deployment_package/cli.py

```diff
@@ -0,0 +1,199 @@
+                                  The criterion to search model for, multiple
+                                  metrics can be specified as comma
+                                  separated values, supported metrics are
+                                  ['accuracy', 'f1', 'recall', 'mAP',
+                                  'latency', 'throughput', 'compression',
+                                  'file_size', 'memory_usage']  [default:
+                                  accuracy]
+  --target [VNNI|DEFAULT]         Deployment target scenario (ie 'VNNI' for
+                                  VNNI capable CPUs)  [default: DEFAULT]
+  --help                          Show this message and exit.  [default:
+                                  False]
+##########
+Examples:
+    1) Fetch the smallest Image Classification Model trained on imagenette
+        sparsezoo.deployment_package --dataset imagenette --optimizing_metric compression
+    2) Fetch the most accurate Image Classification  Model trained on imagenette
+        sparsezoo.deployment_package --dataset imagenette --optimizing_metric accuracy
+    3) Fetch the most performant Question Answering model trained on squad
+        sparsezoo.deployment_package --task qa --dataset squad --optimizing_metric latency
+    4) Fetch the smallest most performant Question Answering model trained on squad
+        sparsezoo.deployment_package --task qa --dataset squad \
+            --optimizing_metric compression \
+            --optimizing_metric accuracy
+"""
+import logging
+from typing import Any, Mapping
+
+import click
+from sparsezoo import Model, deployment_package
+from sparsezoo.utils import (
+    DATASETS,
+    DEFAULT_DEPLOYMENT_SCENARIO,
+    DEFAULT_OPTIMIZING_METRIC,
+    DEPLOYMENT_SCENARIOS,
+    METRICS,
+    TASKS_WITH_ALIASES,
+)
+from sparsezoo.version import __version__
+
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def _csv_callback(ctx, self, value):
+    """
+    A click callback function to parse a comma separated string with metrics
+    into a list
+    """
+    current_metrics = []
+    for metric in value.split(","):
+        metric_ = metric.lower().strip()
+        if metric_ not in METRICS:
+            raise ValueError(f"Specified metric {metric_} is not supported")
+        current_metrics.append(metric_)
+    return current_metrics
+
+
+def _download_deployment_directory(stub: str) -> str:
+    model = Model(stub)
+    model.deployment.download()
+    return model.deployment.path
+
+
+def _get_template(results: Mapping[str, Any]):
+    stub = results.get("stub")
+
+    if not stub:
+        return "No relevant models found for specified metrics"
+
+    stub_info = f"""
+        Relevant Stub: {stub}
+    """
+    metrics = results.get("metrics")
+    metrics_info = (
+        f"""
+        Model Metrics: {metrics}
+        """
+        if metrics
+        else ""
+    )
+    dockerfile_directory = "sparsezoo/src/deployment_package/docker"
```

Comment:
this won't work in built wheel - let's move to `deployement_package/dockerfiles/...` and access via relative path from something like
`deployement_package/docker/helpers.py::DEPLOYMENT_DOCKER_PATH`

additionally will need to add `deployement_package/dockerfiles/*` to `MANIFEST.IN`

Reply:
Moved!